tox.ini

```diff
@@ -1,30 +1,62 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
-envlist = flake8,
-          py27,
-          py35,
-          py36,
-          py37,
-          py38,
-          pypy,
-          pypy3,
-          coverage
+envlist =
+    lint,
+    py27,
+    pypy,
+    py35,
+    py36,
+    py37,
+    py38,
+    pypy3,
+    coverage
 
 [testenv]
 usedevelop = true
+deps =
+    zope.testrunner
 commands =
     zope-testrunner --test-path=src []
 extras = test
 
-[testenv:coverage]
-basepython = python3.7
+[testenv:lint]
+basepython = python3
+skip_install = true
+deps =
+    flake8
+    check-manifest
+    check-python-versions
 commands =
-    coverage run -m zope.testrunner --test-path=src []
-    coverage report --fail-under=86
+    flake8 src setup.py
+    check-manifest
+    check-python-versions
+
+[testenv:coverage]
+basepython = python3
 deps =
     coverage
+    coverage-python-version
+    zope.testrunner
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage html
+    coverage report -m --fail-under=23
```

Comment:
Note that you need special multiprocess coverage magic to compute the coverage correctly for this and other buildout recipes.  See #7.

Reply:
That means we need a special variant in zopefoundation/meta for buildout recipes.

Reply:
See #8 which was built using such a special variant.